week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,97 @@
+//add event listener to blue button
+
+let blueBtn = document.getElementById('blueBtn');
+//console.log(blueEl);
+
+let jumboEl = document.querySelector('.jumbotron');
+//console.log(jumboEl);
+
+let donateBtn = document.getElementById('donateBtn');
+
+let volunteerBtn = document.getElementById('volunteerBtn');
+
+
+
+function onClickBlueBtn(){
+    jumboEl.style.backgroundColor = '#588fbd';
+    donateBtn.style.backgroundColor = '#ffa500';
+    volunteerBtn.style.backgroundColor = '#000000';
```

Comment:
Its also possible to manipulate the style by updating the `classList` property of an element. You can store the styling stuff in a separate file and use methods like `volunteerBtn.classList.add(<insert-class-here>)`